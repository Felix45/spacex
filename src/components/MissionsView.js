import React from 'react';
import { useSelector } from 'react-redux';
import Row from 'react-bootstrap/Row';
import MissionList from './MissionList';

const MissionsView = () => {
  const { items } = useSelector((state) => state.missions);

  return (
    <Row>
      <MissionList missions={items} />
    </Row>
  );
};

export default MissionsView;
