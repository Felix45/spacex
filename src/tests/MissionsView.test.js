import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux';
import MissionsView from '../components/MissionsView';

test('Missions view renders correctly', () => {
  const tree = renderer.create(<Provider store={store({})}><MissionsView /></Provider>).toJSON();
  expect(tree).toMatchSnapshot();
});
