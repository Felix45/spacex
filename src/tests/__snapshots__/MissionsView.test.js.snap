// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions view renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="table-responsive"
  >
    <table
      className="table table-striped table-bordered"
    >
      <thead>
        <tr>
          <th
            width="100"
          >
            Mission
          </th>
          <th>
            Description
          </th>
          <th
            width="100"
          >
            Status
          </th>
          <th
            width="150"
          >
             
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;
